CONTRIBUTING.md

```diff
@@ -20,7 +20,7 @@ connects to the beta server for testing.
 
 If you want to work on Joseki Explorer front-end code, you also need to point to the beta Joseki Explorer server.
 
-Do this by executing `data.set("joseki-url", "https:beta.online-go.com/godojo/")` in the javascript console of your browser,
+Do this by executing `data.set("joseki-url", "https:beta.online-go.com/oje/")` in the javascript console of your browser,
```

Comment:
It might be possible to take the onus off the developer by checking current environment.  If you're interested, I think the global variable to check would be `window.location.hostname` à la AI review:

https://github.com/online-go/online-go.com/blob/c0c6093a0343041bb75ca8843cadc5121f3c3570/src/lib/sockets.ts#L48-L59